autovalet_husky/launch/bagfile.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+
+
+<!-- This launch file runs RTAB and a bagfile in the proper order with the proper delays necessary for
+everything to run properly. See timed_roslaunch.sh for more info. Make sure to chmod +x timed_roslaunch.sh -->
+
+<launch>
+
+    <param name="use_sim_time" value="true"/>
+
+    <!-- what bagfile you want to run (make sure to put it in the autovalet_husky/bags folder)-->
+    <arg name="bagfile" default="costmap_testing_2020-10-07-18-02-13.bag"/>
+
+    <!-- RVIZ -->
+    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find autovalet_husky)/rviz/config.rviz" />
```

Comment:
we really need a discussion on this... if we're removing every RVIZ file from the repo, lines like this should be removed from launch files as well?

Reply:
what i was thinking was that we just remove the "args". that way it'll just call your default rviz.

Reply:
sounds good to me